go.mod

```diff
@@ -17,3 +17,5 @@ require (
 	google.golang.org/genproto v0.0.0-20201109203340-2640f1f9cdfb // indirect
 	google.golang.org/grpc v1.33.2 // indirect
 )
+
+replace github.com/heimweh/go-pagerduty => ../go-pagerduty
```

Comment:
https://github.com/heimweh/go-pagerduty/pull/131 was merged, so you can now proceed to update this dependency and rebase onto `master` @jedelson-pagerduty